src/utils/Processing.ts

```diff
@@ -560,7 +560,11 @@ export function getLocalSushiVersion(): string {
 
 export async function getLatestSushiVersion(): Promise<string> {
   try {
-    const res = await axiosGet('https://registry.npmjs.org/fsh-sushi');
+    const registryRoot = process.env['NPM_CONFIG_REGISTRY']
+      ? process.env['NPM_CONFIG_REGISTRY']
+      : 'https://registry.npmjs.org';
```

Comment:
I expect this will end up changing with your update, but note that I think this can be simplified to:
```suggestion
    const registryRoot = process.env['NPM_CONFIG_REGISTRY'] ?? 'https://registry.npmjs.org';
```